src/index.less

```diff
@@ -3,6 +3,10 @@
 @import '~nav-frontend-core/less/_mixins';
 @import '~nav-frontend-core/less/utilities';
 
+// Manuelle imports. Importert direkte siden komponentene som trekker inn stylingen pr nå ikke er i bruk
+@import '~nav-frontend-lenkepanel-style';
+@import '~nav-frontend-knapper-style';
```

Comment:
Hvor brukes denne stylingen? Kan det importeres der isteden?

Reply:
Tja, problemet er at du kan ikke ha det i en module.less fil siden stylingen ikke vil bli importert, så den må ligge i en .less fil. Følte det ble mer ryddig å samle dette i 1 fil enn å ha flere .less filer med 1 linje for import

Reply:
Kan det ikke importeres i ts fila som bruker den da?

Reply:
Tror jeg har sett det har vært gjort før

Reply:
Det er mulig, men det herper rekkefølgen på stylingen ganske bra :/
Hvis man laster inn less filen i .ts så må man override all stylingen med !important eller med mer spesifikk css.
Gjorde en test nå nettopp.

Reply:
Funker fint hos meg, hvis jeg importerer i SpaLenkepanel

Reply:
Det fungerer, men den laster inn rekkefølgen på LESS filene i en litt kjip rekkefølge. Hvis man laster det inn i .ts og man ønsker å gjøre overskrivinger slik som i menyen på knappen som ligger der. Hvis man ikke endrer på stylingen fra nav-frontend så fungerer det helt greit

Reply:
Flyttet `import 'nav-frontend-lenkepanel-style';` inn i .ts siden vi ikke gjør noen endringer på stylingen og fjernet `import 'nav-frontend-knapper-style';` siden stylingen blir nå trukket inn av kopiering-knappen